src/pages/StandardReport.js

```diff
@@ -33,7 +33,7 @@ class StandardReport extends React.Component {
 
     componentDidMount() {
         if (this.props.reports.length === 0) {
-            this.props.loadStandardReports(true)
+            this.props.loadStandardReports()
```

Comment:
Ok, so I'm assuming that this was the culprit eventually? As it seems that if this is `true` (in the old code), that for versions above minor 34 will trigger a fetch to the `visualization` endpoint (called `visualizations` in the bug report):

https://github.com/dhis2/reports-app/blob/master/src/utils/backwardCompatability.js#L44-L45

Reply:
Yeah that was one of the culprits, the were actually two, see  57023f2 and 4e32382

Reply:
Ah ok, I see, thanks 👍